### ISSUE TITLE
Not working with Scala 2.12 processes

### ISSUE THREAD

**Issue by smeagol74** [xitrum-framework/scalive#19]
Scalive wont working with Scala 2.12 applications

It lists processes, and even shows REPL attached to it, but when trying to do some completion or run a line it just sleep and do nothing. No typing anything also.

**Comment by ngocdaothanh** [comment]
Can you manually download:

```
scala-library-2.12.1.jar
scala-compiler-2.12.1.jar
scala-reflect-2.12.1.jar
```

and put them in the same directory with Scalive?

See [README](https://github.com/xitrum-framework/scalive/blob/master/README.md).

**Comment by smeagol74** [comment]
Yep, same result. I think the issue is with ILoopWithCompletion. It tries to use scala.tools.nsc.interpreter.JLineCompletion wich is not available in scala 12 anymore.

**Comment by ngocdaothanh** [comment]
Thanks for the investigation, I'll investigate more.

**Comment by ngocdaothanh** [comment]
I've fixed the problem, and released new version 1.7.0, see the updated README:
https://github.com/xitrum-framework/scalive
